handlers/bot_handler.py

```diff
@@ -18,11 +18,29 @@ def execute(self, slack_wrapper, args, channel_id, user_id):
         """Announce the bot's presence in the channel."""
         slack_wrapper.post_message(channel_id, "Pong!")
 
+class IntroCommand(Command):
+    """Show an introduction message for new members."""
+
+    def execute(self, slack_wrapper, args, channel_id, user_id):
+        """Execute the Intro command."""
+        try:
+            with open("intro_msg") as f:
```

Comment:
Tested within docker? works with cwd set by docker?

Reply:
Got me on this, only tested it from venv. But just checked, and it also finds it from within docker.